pom.xml

```diff
@@ -20,6 +20,10 @@
     <artifactId>doc-samples</artifactId>
     <packaging>pom</packaging>
 
+    <prerequisites>
```

Comment:
You might remove this.  Things get weird with and w/o it.

Reply:
Yea - I think what we need to do is set the versions of the maven plugins to use.